.github/workflows/test_python.yml

```diff
@@ -17,7 +17,11 @@ jobs:
         run: pip install -e .[testing,notebook]
       - name: TODO - When 3.6 is dropped, just install the normal way.
         run: pip install coverage
-      - name: Run tests
-        run: coverage run --module pytest
-      - name: Report coverage
+      - name: Run tests via coverage with omits
+        run: coverage run --rcfile .coveragerc_omit --module pytest
+      - name: Report coverage with omits
         run: coverage report --show-missing --fail-under 100
+      - name: Run tests via coverage without omits
+        run: coverage run --rcfile .coveragerc_real --module pytest
```

Comment:
I could be missing something, but I don't think this will include the blocks marked with `pragma: no cover`: For example, there's one line in `create_test_data.py`:
```
    except ModuleNotFoundError:  # pragma: no cover
        # TODO: When we don't need backward compatibility, move import back to top.
        return
```
Coverage says 100%:
```
$ coverage report --show-missing | grep create_test_data
tests/create_test_data.py      36      0   100%
```
but if I take out that comment ...
```
$ git diff
...
-    except ModuleNotFoundError:  # pragma: no cover
+    except ModuleNotFoundError:
...
```
and rerun:
```
$ coverage report --show-missing | grep create_test_data
tests/create_test_data.py      38      2    95%   151-153
```

Here's what I think would work: Use [`exclude_lines`](https://coverage.readthedocs.io/en/4.5.4/excluding.html#advanced-exclusion) with an _empty_ set of patterns in `.coveragerc_real`

Reply:
Thanks, I figured the #pragma: no cover lines would make the numbers slightly inaccurate but didn't know about the `exclude_lines` trick. I just made a commit to add that